admins/pageflow/entry.rb

```diff
@@ -49,7 +49,9 @@ module Pageflow
     end
 
     filter :title
-    filter :account
+    filter :account,
+           if: ->(_) { authorized?(:index, :accounts) },
+           collection: -> { indexable_accounts }
```

Comment:
As long as we are depending on `cancancan` this seems like the perfect use case for `Account.accessible_by(current_ability)`. Should we ever switch to `pundit`, it would become `pundit_scope(Account)`. That way we can skip the helper. I'd also skip `indexable` since that basically is what `resolve` is used for in the context of Pundit/ActiveAdmin. The helper also does not use the alias at moment.

Since we are coding some logic in these ActiveAdmin DSL calls, I think it would have been justified to add a controller spec that checks the `select` tag inside the rendered view. That way we would be sure that nothing breaks during ActiveAdmin updates. But since I'm sure there are many cases where we haven't done it either, I'm also ok with leaving it untested.